science/gmt5/Portfile

```diff
@@ -8,8 +8,8 @@ version             5.4.5
 revision            1
 conflicts           gmt6
 subport gmt6 {
-    version         6.0.0rc5
-    revision        1
+    version         6.0.0
+    revision        0
```

Comment:
In cases like this, port sees `6.0.0rc5` as newer than `6.0.0` and so the `epoch` line needs to be added. 

`epoch   1
`